File: src/main/java/com/github/pauldambra/moduluschecker/As.java
package com.github.pauldambra.moduluschecker;

import java.util.stream.Stream;

public final class As {
    public static Stream<Integer> integerStream(String s) {
        return s.chars()
                .mapToObj(i -> (char) i)
                .map(String::valueOf)
                .map(Integer::parseInt);
    }
}

File: src/main/java/com/github/pauldambra/moduluschecker/As.kt
package com.github.pauldambra.moduluschecker

import java.util.stream.Stream

object As {
    fun integerStream(s: String): Stream<Int> {
        return s.chars()
          .mapToObj { i -> i.toChar() }
          .map<String> { it.toString() }
          .map { Integer.parseInt(it) }
    }
}
